### PR TITLE
Update ck and enable test

### PR DESCRIPTION
**Pull request by cloudhan**
Apply the fix in https://github.com/ROCmSoftwarePlatform/composable_kernel/issues/728

### CODE REVIEW

**Comment by snnn** [comment]
Is this PR still needed?

**Comment by cloudhan** [comment]
I don't know what makes you think it is not needed? Elobrate please? @snnn

File: tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml [hunk 1]
```diff
@@ -176,13 +176,14 @@ jobs:
           --user onnxruntimedev \
           --volume $(Build.SourcesDirectory):/onnxruntime_src \
           --volume $(Build.BinariesDirectory):/build \
+          -e OPENBLAS_NUM_THREADS=1 \
```
Comment (PeixuanZuo) [inline tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml:179]: Could you please add comment about these new  env variables in PR description?

**Review by PeixuanZuo — APPROVED** [review]
LGTM.

File: cmake/external/composable_kernel.cmake [hunk 2]
```diff
@@ -1,5 +1,5 @@
 set(composable_kernel_URL https://github.com/ROCmSoftwarePlatform/composable_kernel.git)
-set(composable_kernel_TAG ed3a2e52265e11daa366f47b082141a652b67c58) # 2023-04-10 21:02:17 +0800
+set(composable_kernel_TAG d52ec01652b7d620386251db92455968d8d90bdc) # 2023-08-18 11:14:59 +0800
```
Comment (snnn) [inline cmake/external/composable_kernel.cmake:2]: Later on please move it to deps.txt

**Review by snnn — APPROVED** [review]

**Pull request merged**